src/shared/nt-core/withLoader.styl

```diff
@@ -0,0 +1,38 @@
+@require('../nt-design/colorsets')
+
+l-size = 5em
+l-border = 5px
+l-spin-time = 1000ms
+
+@keyframes spinAround 
+    from 
+        transform rotate(0deg)
+    to
+        transform rotate(360deg)
+
+.loader
+    display flex
+    flex-direction row
+    justify-content center
+    align-items center
+    position absolute
+    top 0
+    left 0
+    height 100%
+    width 100%
+
+    &:before
+        -webkit-animation spinAround l-spin-time infinite linear
+        animation spinAround l-spin-time infinite linear
+        border l-border solid c-gray50
+        border-top l-border solid c-gray500
+        border-radius 50%
+        content ""
+        display block
+        height l-size
+        width l-size
+        margin-top -5em
+    
+    p
+        position absolute
+        font-size 2rem
```

Comment:
worng indentation (2 instead of 4) and please check the stylint especially for css order
https://github.com/SimenB/stylint/blob/master/src/data/ordering.json

Anyway, leaveing css order is ok since it is time consuming, but please fix the indentation